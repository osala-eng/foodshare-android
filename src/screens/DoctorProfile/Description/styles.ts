import { StyleSheet } from 'react-native';
import spacing from '../../../theme/spacing';

export default StyleSheet.create({
  container: {
    marginTop: spacing.medium1,
    paddingHorizontal: spacing.medium3,
  },
  title: {
    marginBottom: spacing.small1,
  },
});
