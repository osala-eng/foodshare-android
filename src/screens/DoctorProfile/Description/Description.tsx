import React from 'react';
import { View } from 'react-native';
import Text from '../../../components/Text';
import styles from './styles';

interface Props {
  title: string;
  details: string;
}
const Description = ({ title, details }: Props) => {
  return (
    <View style={styles.container}>
      <Text color="black" size="head-24" style={styles.title}>
        {title}
      </Text>
      <Text color="dark" size="body-16" selectable>
        {details}
      </Text>
    </View>
  );
};

export default Description;
