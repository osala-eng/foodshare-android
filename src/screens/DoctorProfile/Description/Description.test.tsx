import React from 'react';
import { render } from '@testing-library/react-native';
import Description from './Description';

describe('Button component test', () => {
  it('renders correctly', () => {
    const { toJSON } = render(
      <Description
        title="Title"
        details="Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa, autem."
      />,
    );

    expect(toJSON()).toMatchSnapshot();
  });
});
