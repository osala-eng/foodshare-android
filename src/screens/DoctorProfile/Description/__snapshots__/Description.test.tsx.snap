// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component test renders correctly 1`] = `
<View
  style={
    Object {
      "marginTop": 20,
      "paddingHorizontal": 32,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#000000",
        "fontFamily": "System",
        "fontSize": 24,
        "fontWeight": "400",
        "includeFontPadding": false,
        "lineHeight": 30,
        "marginBottom": 10,
        "textAlign": undefined,
      }
    }
    testID="Text"
  >
    Title
  </Text>
  <Text
    selectable={true}
    style={
      Object {
        "color": "#a6aaad",
        "fontFamily": "System",
        "fontSize": 16,
        "fontWeight": "400",
        "includeFontPadding": false,
        "lineHeight": 20,
        "textAlign": undefined,
      }
    }
    testID="Text"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa, autem.
  </Text>
</View>
`;
